Tests/Indicators/DeltaTests.cs

```diff
@@ -141,25 +140,23 @@ public void ComparesDeltaOnBSMModel(decimal price, decimal spotPrice, OptionRigh
         }
 
         // Reference values from QuantLib
-        [TestCase(23.753, 450.0, OptionRight.Call, 60, 0.5432)]
-        [TestCase(35.830, 450.0, OptionRight.Put, 60, -0.4494)]
-        [TestCase(33.928, 470.0, OptionRight.Call, 60, 0.6884)]
-        [TestCase(6.428, 470.0, OptionRight.Put, 60, -0.2650)]
-        [TestCase(3.219, 430.0, OptionRight.Call, 60, 0.2419)]
-        [TestCase(47.701, 430.0, OptionRight.Put, 60, -0.5301)]
-        [TestCase(16.528, 450.0, OptionRight.Call, 180, 0.6162)]
-        [TestCase(21.784, 450.0, OptionRight.Put, 180, -0.4392)]
-        [TestCase(35.207, 470.0, OptionRight.Call, 180, 0.7461)]
-        [TestCase(0.409, 470.0, OptionRight.Put, 180, -0.0606)]
-        [TestCase(2.642, 430.0, OptionRight.Call, 180, 0.2579)]
-        [TestCase(27.772, 430.0, OptionRight.Put, 180, -0.6020)]
+        [TestCase(23.753, 450.0, OptionRight.Call, 60, 0.546)]
+        [TestCase(35.830, 450.0, OptionRight.Put, 60, -0.446)]
+        [TestCase(33.928, 470.0, OptionRight.Call, 60, 0.693)]
+        [TestCase(6.428, 470.0, OptionRight.Put, 60, -0.260)]
+        [TestCase(3.219, 430.0, OptionRight.Call, 60, 0.243)]
+        [TestCase(47.701, 430.0, OptionRight.Put, 60, -0.526)]
+        [TestCase(16.528, 450.0, OptionRight.Call, 180, 0.632)]
+        [TestCase(21.784, 450.0, OptionRight.Put, 180, -0.417)]
+        [TestCase(35.207, 470.0, OptionRight.Call, 180, 0.765)]
+        [TestCase(0.409, 470.0, OptionRight.Put, 180, -0.052)]
+        [TestCase(2.642, 430.0, OptionRight.Call, 180, 0.264)]
+        [TestCase(27.772, 430.0, OptionRight.Put, 180, -0.556)]
         public void ComparesDeltaOnCRRModel(decimal price, decimal spotPrice, OptionRight right, int expiry, double refDelta)
         {
             // Under CRR framework
             var symbol = Symbol.CreateOption("SPY", Market.USA, OptionStyle.American, right, 450m, _reference.AddDays(expiry));
-            var indicator = new Delta(symbol, 0.0530m, 0.0153m,
-                    optionModel: OptionPricingModelType.BinomialCoxRossRubinstein,
-                    ivModel: OptionPricingModelType.BlackScholes);
+            var indicator = new Delta(symbol, 0.0403m, 0.0m, OptionPricingModelType.BinomialCoxRossRubinstein, OptionPricingModelType.BlackScholes);
```

Comment:
There's no difference with the unit tests above arguments, should there be? Seems they could be joined or share a main implementation method